test-e2e/cip-auditor/cip-auditor-e2e.go

```diff
@@ -299,96 +299,28 @@ func testSetup(
 		return err
 	}
 
-	if err := populateGoldenImages(repoRoot, pushRepo); err != nil {
+	if err := populateGoldenImages(repoRoot); err != nil {
 		return err
 	}
 
 	return nil
 }
 
 // nolint[funlen]
```

Comment:
Non-blocking: let's drop this in a follow-up.

Reply:
Nice catch.

Reply:
Great catch! Changing now.